src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/BuildPropertyPage/DefineConstantsVBValueProvider.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.ProjectSystem.Debug;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+[ExportInterceptingPropertyValueProvider(DefineConstantsPropertyName, ExportInterceptingPropertyValueProviderFile.ProjectFile)]
+[AppliesTo(ProjectCapability.VisualBasic)]
+internal sealed class DefineConstantsVBValueProvider : InterceptingPropertyValueProviderBase
+{
+    internal const string DefineConstantsPropertyName = "DefineConstants";
```

Comment:
this can be taken as a constant from GeneralBrowseObject

Reply:
Should probably take the property from `ConfigurationGeneral` which is intended for programmatic access, while the browse object is for the UI.

```suggestion
    internal const string DefineConstantsPropertyName = ConfigurationGeneral.DefineConstantsProperty;
```